src/components/pages/EditBirb/EditBirb.js

```diff
@@ -1,207 +1,187 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import _ from 'underscore';
 import DatePicker from 'react-datepicker';
 
 import birbsData from '../../../helpers/data/birbsData';
 
-class EditBirb extends React.Component {
-  state = {
-    birb: {
-      type: '',
-      color: '',
-      size: '',
-      seenAt: new Date(),
-      altColor: '',
-      wasSleeping: true,
-      location: '',
-      notes: '',
-    },
-  }
-
-  componentDidMount() {
-    birbsData.getBirbById(this.props.match.params.birbId)
+const EditBirb = (props) => {
+  const [birb, setBirb] = useState({
+    type: '',
+    color: '',
+    size: '',
+    seenAt: new Date(),
+    altColor: '',
+    wasSleeping: true,
+    location: '',
+    notes: '',
+  });
+
+  useEffect(() => {
+    birbsData.getBirbById(props.match.params.birbId)
       .then((res) => {
-        const birb = res.data;
-        birb.seenAt = new Date(birb.seenAt);
-        this.setState({ birb });
+        const bord = res.data;
+        bord.seenAt = new Date(bord.seenAt);
+        setBirb(bord);
       })
       .catch((err) => console.error('err in get single birb', err));
-  }
+  }, [props.match.params.birbId]);
 
-  changeTypeEvent = (e) => {
-    e.preventDefault();
-    const { birb } = this.state;
-    birb.type = e.target.value;
-    this.setState({ birb });
-  }
+  const changeTypeEvent = (e) => {
+    const birbCopy = _.clone(birb);
```

Comment:
probably could use the `...` operator here instead of underscore, but I already had underscore soooo

reason I need this is because I don't want to change the `birb` state item directly while I'm editing it...it gets weird. So I make a copy to set into state